src/demo/java/org/junitpioneer/jupiter/DisableIfTestFailsExtensionDemo.java

```diff
@@ -38,7 +38,7 @@ void test1() {
 		@Test
 		@Order(2)
 		void test2() {
-			fail();
```

Comment:
Below you use `""`? (Too bad AssertJ has no no-arg `fails`.)

Reply:
Not yet but maybe soon 😉 assertj/assertj#3204